commit e608d4f7fe1a2085b22af206d0c8c2cc128c1e9a
Author: Martin Kroeker <martin@ruby.chemie.uni-freiburg.de>
Date:   Wed Mar 13 22:10:28 2019 +0100

    Disable the AVX512 DGEMM kernel (again)
    
    Due to as yet unresolved errors seen in #1955 and #2029

diff --git a/kernel/x86_64/KERNEL.SKYLAKEX b/kernel/x86_64/KERNEL.SKYLAKEX
index acc6356..5d0a300 100644
--- a/kernel/x86_64/KERNEL.SKYLAKEX
+++ b/kernel/x86_64/KERNEL.SKYLAKEX
@@ -7,7 +7,7 @@ SGEMMITCOPY    =  sgemm_tcopy_16_skylakex.c
 SGEMMONCOPY    =  sgemm_ncopy_4_skylakex.c
 SGEMMOTCOPY    =  ../generic/gemm_tcopy_4.c
 
-DGEMMKERNEL    =  dgemm_kernel_4x8_skylakex.c
+#DGEMMKERNEL    =  dgemm_kernel_4x8_skylakex.c
 
 DGEMMINCOPY    =  dgemm_ncopy_8_skylakex.c
 DGEMMITCOPY    =  dgemm_tcopy_8_skylakex.c
